internal/graph/dispatch_throttling_check_resolver_test.go

```diff
@@ -0,0 +1,105 @@
+package graph
+
+import (
+	"context"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+	"go.uber.org/goleak"
+)
+
+type countResolveCheck struct {
+	count *atomic.Uint32
+}
+
+func newCountResolveCheck() *countResolveCheck {
+	return &countResolveCheck{
+		count: &atomic.Uint32{},
+	}
+}
+
+func (c *countResolveCheck) ResolveCheck(ctx context.Context, req *ResolveCheckRequest) (*ResolveCheckResponse, error) {
+	c.count.Add(1)
+	return &ResolveCheckResponse{Allowed: true}, nil
+}
+
+func (c *countResolveCheck) Close() {}
+
+func (c *countResolveCheck) NumResolveCheck() uint32 {
```

Comment:
It feels odd to wrap a single function call `c.count.Load()` into a function. Can we just `c.count.Load()` where needed instead?

Reply:
obsolete when we use the MockCheckResolver